src/components/expanded-state/custom-gas/FeesGweiInput.tsx

```diff
@@ -1,69 +1,80 @@
 import React, { useCallback, useEffect, useRef, useState } from 'react';
 import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
 import { ButtonPressAnimation } from '../../animations';
-import { Row } from '../../layout';
-import { Text } from '../../text';
 import GweiInputPill from './GweiInputPill';
 import { delay } from '@/helpers/utilities';
 import { usePrevious } from '@/hooks';
-import styled from '@/styled-thing';
+import { TextInput } from 'react-native';
+import { Box, Inline, Text } from '@/design-system';
+import { colors } from '@/styles';
 
 const PLUS_ACTION_TYPE = 'plus';
 const MINUS_ACTION_TYPE = 'minus';
 const LONG_PRESS_DELAY_THRESHOLD = 69;
 const MIN_LONG_PRESS_DELAY_THRESHOLD = 200;
 
-const Wrapper = styled(Row)({});
-
-const StepButtonWrapper = styled(ButtonPressAnimation).attrs(() => ({
-  paddingHorizontal: 7,
-  scaleTo: 0.75,
-}))({});
-
-const StepButton = styled(Text).attrs(({ theme: { colors }, color }) => ({
-  color: color || colors.appleBlue,
-  lineHeight: 40,
-  size: 'lmedium',
-  weight: 'heavy',
-}))({});
-
 const GweiStepButton = ({
-  type,
+  buttonColor,
   onLongPress,
   onLongPressEnded,
   onPress,
+  type,
   shouldLongPressHoldPress,
-  buttonColor,
+}: {
+  buttonColor: string;
+  onLongPress: () => void;
+  onLongPressEnded: () => void;
+  onPress: () => void;
+  type: string;
+  shouldLongPressHoldPress: boolean;
 }) => {
   return (
-    <StepButtonWrapper
+    <Box
+      as={ButtonPressAnimation}
+      // @ts-expect-error
+      scaleTo={0.75}
       minLongPressDuration={MIN_LONG_PRESS_DELAY_THRESHOLD}
       onLongPress={onLongPress}
       onLongPressEnded={onLongPressEnded}
       onPress={onPress}
       shouldLongPressHoldPress={shouldLongPressHoldPress}
       useLateHaptic={false}
+      padding="4px"
+      margin="-4px"
```

Comment:
Why are we using the negative margin here?

Reply:
i did it in order to have a bigger surface to press, without it the only area to press would be the text, with this is the text plus 4px around